packages/desktop-electron/index.js

```diff
@@ -92,28 +99,6 @@ function createBackgroundProcess(socketName) {
   });
 }
 
-function createBackgroundWindow(socketName) {
```

Comment:
Is it still possible to open a devtools window for the server process? It can be helpful to be able to debug it sometimes.

Reply:
I can in vscode, but if you're running in the terminal you will find it difficult.

The issue was that this background window didnt actually have a toolbar, so you couldn't click `View -> Open Developer Tools`, and the ctrl+shift+i shortcut didnt work either, so you didn't actually have devtools for the server process anyway. 

I tried forcing the window to have a toolbar but this removed the toolbar from the main window. Someone with more electron experience might be able to solve this.